draft-gruessing-ntp-ntpv5-requirements.md

```diff
@@ -154,30 +154,29 @@ timescale in use.
 ## Leap seconds
 
 Tranmission of UTC leap second information MUST be included in the protocol in
-order for clients to generate a UTC representation but must be transmitted as
-separate information to the timescale. The specification SHOULD also be capable
+order for clients to generate a UTC representation, but must be transmitted as
+separate information to the timescale. The specification SHOULD be capable
 of transmitting upcoming leap seconds greater than 1 calendar day in advance.
 
 Leap second smearing SHOULD NOT be applied to timestamps transmitted by the
 server, however this should not prevent implementers from applying leap second
 smearing between the client and any clock it is training.
 
-## Backwards compatibility to NTS and NTPv4
+## Backwards compatibility with NTS and NTPv4
 
-The support for compatibility with other protocols should not prevent addressing
-issues that have previous caused issues in deployments or cause ossification of
-the protocol.
+The desire for compatibility with older protocols should not prevent addressing
+deployment issues or cause ossification of the protocol.
+
+The model for backward compatibility is: servers that support multiple versions
+of NTP must send a response in the same version as the request. This does not
+preclude servers from acting as a client in one version of NTP and
+a server in another.
```

Comment:
I'm not sure the inclusion of a colon here improves readability. This could be rewritten to remove the whole "The model for backward compat...", maybe something like:

```suggestion
Servers that support multiple versions of NTP must respond in the same version as the request it received. This does not preclude servers using different NTP versions when acting as a client to lower stratum servers. 
```
<sub>(and later I'll think if normalising that "must" is needed)</sub>